airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/ReplicationActivityImpl.java

```diff
@@ -53,7 +65,11 @@
   private final String databaseUrl;
   private final String airbyteVersion;
 
-  public ReplicationActivityImpl(
+  private static final MdcScope.Builder LOG_MDC_BUILDER = new MdcScope.Builder()
+      .setLogPrefix("replication-runner")
+      .setPrefixColor(LoggingHelper.Color.BLACK);
```

Comment:
This won't be Highlighted and will looks like a regular log, Should we add more color code. I also want to suggest to do a dark/accessibility mode for the logs. I wonder if all the color that we picked are visible for the colorblind people. I believe that having a higher contrast will make it easier to read and in this case if the background is black we won't see the prefix.

Reply:
Fixed.